proposals/serverlessworkflow.md

```diff
@@ -0,0 +1,185 @@
+# Serverless Workflow
+
+**Name of project**: Serverless Workflow
+
+**Description**:
+
+TOC created the Serverless Working Group to investigate
+the Serverless landscape. The outputs of the WG included:
+
+- a [whitepaper](https://github.com/cncf/wg-serverless/blob/master/whitepapers/serverless-overview/README.md) that:
+  - defines Serverless and its terminology
+  - describes common use cases for the technology
+  - compares it with other Cloud Native technologies and \*aaS environments
+  - describes the common architecture of Serverless platforms
+- a [landscape document](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0)
+  that lists well-known open-source and proprietary Serverless platforms
+  and tools
+- a set of recommended next steps for the WG, as part of the whitepaper:
+  - encourage more Serverless technology vendors and open source developers
+    to join the CNCF
+  - foster an open ecosystem by establishing interoperable APIs, in particular
+    around: Events, Deployments and Workflows
+  - provide additional education as needed
+
+The Working Group's first focus area was around making it easier for middleware to process Events,
+which led to the development of the [CloudEvents](https://cloudevents.io) specification.
+
+Additionally, the Serverless WG has agreed to form a sub working group to work on
+the Serverless Workflow specification.
+
+Serverless Workflow is a vendor-neutral specification for defining the model of workflows responsible for 
+orchestrating event-driven serverless applications.
+
+Goals of the Serverless Workflow sub-group are to :
+
+- Standardize Serverless Workflow model and definition
+- Facilitate Serverless Workflow portability
+- Be completely vendor neutral
+- Support both stateless and stateful Serverless Workflow implementations
+- To define a light-weight and powerful Serverless Workflow model
+
+Functional scope of the Serverless Workflow specification includes:
+
+- Define and orchestrate steps/states involved in a serverless application.
+- Define which functions are executed in each step.
+- Define which event or combination of events trigger function execution.
+- Define function execution behavior (sequential, parallel, etc).
+- Specify information filtering throughout the execution of the serverless workflow.
+- Define error conditions with retries.
+- Define correlation between events and workflow instances.
+- Specify workflow integration with third-party services.
+
+Here is diagram that illustrates the Serverless Workflow functional flow:
+
+<p align="center">
+<img src="https://github.com/cncf/wg-serverless/raw/master/workflow/spec/media/sample-serverless-workflow2.png" with="400px" height="260px" alt="Serverless Workflow Diagram"/>
+</p>
+
+You can see the Serverless Workflow specification examples
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/spec-examples.md).
```

Comment:
broken link?

Reply:
Nice catch! we recently updated doc naming conventions so some links changed. I updated all the links as well as the path to the serverless workflow diagram included in the proposal. Let me know if you run into anything again. Thanks!